pkg/datapath/orchestrator/orchestrator.go

```diff
@@ -36,10 +47,32 @@ func newOrchestrator(params orchestratorParams) *orchestrator {
 }
 
 func (o *orchestrator) Reinitialize(ctx context.Context) error {
+	// Wait until the local node has been populated by NodeDiscovery.
+	o.params.NodeDiscovery.WaitForLocalNodeInit()
+
+	localNode, err := o.params.LocalNodeStore.Get(ctx)
+	if err != nil {
+		return fmt.Errorf("get local node: %w", err)
+	}
+
+	// Construct the LocalNodeConfiguration that encapsulates the
+	// local node's dynamic configuration.
+	localNodeConfig, err := newLocalNodeConfig(
+		option.Config,
+		localNode,
+		o.params.MTU,
+		o.params.DB.ReadTxn(),
+		o.params.Devices,
```

Comment:
I might have missed it, but is there anything that writes to this table (or the previous one) at runtime (after populating the Node config)? In other words, it looks like this orchestrator would now be the place to record entirely new devices, and I'm not sure if anything currently needs to do that.

Reply:
Ah, I found the calls to `Orchestrator().Reinitialize()`, nice